src/workerd/api/wpt/url-test.js

```diff
@@ -2,140 +2,136 @@
+      'request.formData() with input: a&b&c',
+      'response.formData() with input: a&b&c',
+      'request.formData() with input: a=b&c=d',
+      'response.formData() with input: a=b&c=d',
+      'request.formData() with input: a=b&c=d&',
+      'response.formData() with input: a=b&c=d&',
+      'request.formData() with input: &&&a=b&&&&c=d&',
+      'response.formData() with input: &&&a=b&&&&c=d&',
+      'request.formData() with input: a=a&a=b&a=c',
+      'response.formData() with input: a=a&a=b&a=c',
+      'request.formData() with input: a==a',
+      'response.formData() with input: a==a',
+      'request.formData() with input: a=a+b+c+d',
+      'response.formData() with input: a=a+b+c+d',
+      'request.formData() with input: %=a',
+      'response.formData() with input: %=a',
+      'request.formData() with input: %a=a',
+      'response.formData() with input: %a=a',
+      'request.formData() with input: %a_=a',
+      'response.formData() with input: %a_=a',
+      'request.formData() with input: %61=a',
+      'response.formData() with input: %61=a',
+      'request.formData() with input: %61+%4d%4D=',
+      'response.formData() with input: %61+%4d%4D=',
+      'request.formData() with input: id=0&value=%',
+      'response.formData() with input: id=0&value=%',
+      'request.formData() with input: b=%2sf%2a',
+      'response.formData() with input: b=%2sf%2a',
+      'request.formData() with input: b=%2%2af%2a',
+      'response.formData() with input: b=%2%2af%2a',
+      'request.formData() with input: b=%%2a',
+      'response.formData() with input: b=%%2a',
+    ],
+  },
+  'urlsearchparams-constructor.any.js': {
     expectedFailures: [
       'URLSearchParams constructor, DOMException as argument',
       'Construct with 2 unpaired surrogates (no trailing)',
       'Construct with 3 unpaired surrogates (no leading)',
       'Construct with object with NULL, non-ASCII, and surrogate keys',
     ],
-  }
-);
-export const urlSearchParamsDelete = run('urlsearchparams-delete.any.js');
-export const urlSearchParamsForEach = run('urlsearchparams-foreach.any.js');
-export const urlSearchParamsGetAny = run('urlsearchparams-get.any.js');
-export const urlSearchParamsGetAll = run('urlsearchparams-getall.any.js');
-export const urlSearchParamsHas = run('urlsearchparams-has.any.js');
-export const urlSearchParamsSet = run('urlsearchparams-set.any.js');
-export const urlSearchParamsSize = run('urlsearchparams-size.any.js');
-export const urlSearchParamsSort = run('urlsearchparams-sort.any.js', {
-  expectedFailures: ['Parse and sort: ﬃ&🌈', 'URL parse and sort: ﬃ&🌈'],
-});
-export const urlSearchParamsStringifier = run(
-  'urlsearchparams-stringifier.any.js'
-);
+  },
+  'urlsearchparams-sort.any.js': {
+    expectedFailures: ['Parse and sort: ﬃ&🌈', 'URL parse and sort: ﬃ&🌈'],
+  },
+  'idlharness.any.js': {
+    skipAllTests: true,
+  },
+  // TODO(soon): Implement globalThis.document
+  'a-element-origin.js': {
```

Comment:
Can you enable this lint rule for these kinds of files? https://eslint.org/docs/latest/rules/sort-keys

Reply:
It's all sortitudinated now.